tests/unit/other.tcl

```diff
@@ -332,7 +332,7 @@ start_server {tags {"other external:skip"}} {
         # Hash table should not rehash
         assert_no_match "*table size: 8192*" [r debug HTSTATS 9]
         exec kill -9 [get_child_pid 0]
-        after 200
+        waitForBgsave r
```

Comment:
don't we also need to wait for cron to kick in and start rehasing? maybe it's better to keep the `after` too?

Reply:
ohh right, default hz is 10, so maybe a `after 20` is enough?

Reply:
10hz is 100ms.
let's just keep the 200, and add a comment that it's waiting for serverCron

Reply:
silly me. added it back